scripting/sobekengine/Makefile

```
@@ -0,0 +1,2 @@
+test:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)